pkg/api/v1/routes/handlers.go

```diff
@@ -0,0 +1,36 @@
+package routes
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/metal-toolbox/hollow-bomservice/internal/parse"
+	sservice "go.hollow.sh/serverservice/pkg/api/v1"
+)
+
+func (r *Routes) billOfMaterialsBatchUpload(c *gin.Context) (int, *sservice.ServerResponse) {
+	data, err := io.ReadAll(c.Request.Body)
```

Comment:
We likely want a check on the incoming data size so that a large file (or a series of files that sums to a large size) doesn't blow past our limits in the k8s cluster. Consider rejecting a request if we can't determine the `ContentLength` of the `http.Request` (which is available from the `gin.Context`).